### DISCUSSION TITLE
Can/Should we remove unused imports?

### DISCUSSION THREAD

**Discussion by DelusionArun** [CastleLab/COMP3021-2021Fall-PA1-Student-Version#15] [Q&A] [answered]
After running code style, I get two warnings - Unused import - java.lang.reflect.Array. and Unused import - java.util.ArrayList. because I do not use them in my implementation.

Can I remove these imports or is it better to leave it as it is (since 2 errors do not deduct any marks and they were part of code template)?

**Comment by Derppening** [comment, marked as answer]
If they are unused imports, and you can compile your code without those imports, then you can remove them.
